charts/de-elasticsearch/README.md

```diff
@@ -0,0 +1,8 @@
+Deploys the [official elastic search helm chart](https://github.com/elastic/helm-charts/tree/f01036e04be9aa62afa52f052c4e600c9e3e1da7/elasticsearch)
```

Comment:
It would be more legible to use the tag instead of the hash here. But maybe this is autogenerated? If so, I'd just leave it as is.
```suggestion
Deploys the [official elastic search helm chart](https://github.com/elastic/helm-charts/tree/v7.10.2/elasticsearch)
```